sympy/physics/mechanics/functions.py

```diff
@@ -299,44 +299,16 @@ def potential_energy(*body):
 
 
 def gravity(acceleration, *bodies):
```

Comment:
What should we do with the `gravity` function?
Currently one could only import it from `sympy.physics.mechanics.functions`, so moving it with a deprecation warning will result in a deprecation warning for everyone who was using it (though I don't actually expect that many knew of its existence, it isn't in the docs).
The main advantage is that having it in `loads.py` is more sensible. However there is no necessity due to circular import errors as was the case with the inertia functions.

Reply:
Personally I don't mind relocating it to where it makes most logical sense for it to be located, especially if it hasn't been documented. Given that it could only be imported from `sympy.physics.mechanics.functions`, that to me indicates that it wasn't ever formally part of the `sympy.physics.mechanics` API.

Reply:
Would we like to make it part of the API, personally I don't think it is necessary. A method to apply gravity to the future `System` class seems more useful to me.